base/logging.go

```diff
@@ -284,41 +292,79 @@ func (config *LogAppenderConfig) ValidateLogAppender() error {
 // The key "bw" is interpreted as a call to LogNoColor, not a key.
 func ParseLogFlags(flags []string) {
 	logLock.Lock()
+	keyMap := make(map[string]bool)
+
 	for _, key := range flags {
+		keyMap [key] = true
+	}
+
+	ParseLogFlagsMap(keyMap)
+	logLock.Unlock()
+	Logf("Enabling logging: %s", flags)
+}
+
+// Parses a map of log keys and enabled bool, probably coming from a argv flags.
+// The key "bw" is interpreted as a call to LogNoColor, not a key.
+func ParseLogFlagsMap(flags map[string]bool) {
+	for key, val := range flags {
```

Comment:
What do you think about using 'enabled' instead of 'val' here?  I think it makes the rest of the code below more readable.

Reply:
refactored val to enabled